lib/elixir/lib/enum.ex

```diff
@@ -2435,15 +2434,16 @@ defmodule Enum do
 
   ## uniq
 
-  defp do_uniq([h|t], acc, fun) do
+  defp do_uniq([h|t], set, fun) do
```

Comment:
Sorry for nitpicking, but it is still an accumulator and set is an implementation detail. :bowtie: